nservicebus/messaging/uniformsession.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Uniform Session between message session and pipeline
+summary: Uniform Session allows to bridge the gap between the message session and the pipeline context and enables dependency injection.
+reviewed: 2017-10-27
+component: UniformSession
+---
+
+NServiceBus v6 and higher introduced a few design changes that how code that is using previous versions is structured. The [changes](/nservicebus/upgrades/5to6/moving-away-from-ibus.md) are:
```

Comment:
> [...] design changes that how code that is using previous versions [...]

doesn't read very nicely imo. Maybe something like:
NServiceBus Version 6 introduced significant design changes on how to send messages. Here's a rough outline of the changes: [...]